.scalafmt.conf

```diff
@@ -1,20 +1,27 @@
-maxColumn = 80
-
-style = IntelliJ
-danglingParentheses = true
-indentOperator = spray
+align = none
```

Comment:
Probably after this bug is solved: https://github.com/scalameta/scalafmt/issues/994?